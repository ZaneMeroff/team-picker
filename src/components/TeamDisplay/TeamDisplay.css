.team-display-outer-container {
  height: auto;
  width: 200px;
  padding: 20px;
  border: 2px solid black;
}
