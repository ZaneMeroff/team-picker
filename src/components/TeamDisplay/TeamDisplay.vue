<template>
  <h1>team dispaly</h1>
</template>

<script>
  export default {
    name: 'TeamDisplay',
    props: {},
    data() {
      return {
        // code
      }
    },
    methods: {}
  }
</script>

<style>
  @import './TeamDisplay.css';
</style>
