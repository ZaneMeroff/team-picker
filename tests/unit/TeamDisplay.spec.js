import { shallowMount } from '@vue/test-utils'
import TeamDisplay from '../../src/components/TeamDisplay/TeamDisplay.vue'

describe('TeamDisplay.vue', () => {

  let wrapper;

  beforeEach(() => {
    wrapper = shallowMount(TeamDisplay, {
      propsData: {
        id: 1,
        playerNames: ['','']
      }
    })
  })

  it('component render matches snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })

  it('component has correct props', () => {
    expect(wrapper.props().id).toEqual(1)
    expect(wrapper.props().playerNames).toEqual(['',''])
  })

})
