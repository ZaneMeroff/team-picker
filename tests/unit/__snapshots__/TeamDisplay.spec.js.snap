// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TeamDisplay.vue component renders and matches snapshot 1`] = `
<div class="team-display-outer-container">
  <p class="team-text">team #NaN</p>
</div>
`;
